### DISCUSSION TITLE
Any way to whitelist keepassxc?

### DISCUSSION THREAD

**Discussion by VlKlNGS** [keepassxreboot/keepassxc#11239] [Q&A] [answered]
Hello World,

Is there any way to “whitelist” keepassxc or something so I won’t get the prompt below almost every time I try to use the auto-type feature? Especially since often times clicking share will freeze my system for 30 secs or so… :cry:

I'm using arch Linux(EndeavourOS with KDE Plasma to be exact) and the Brave browser, let me know if you need anything else.

Thank you.

![a71d3ff0cef79c0f4d0de4f32d635b511850edbb](https://github.com/user-attachments/assets/6d9c213e-ae8d-440d-beb1-84a8b815f815)

**Comment by droidmonkey** [comment]
https://github.com/keepassxreboot/keepassxc/issues/2281#issuecomment-2242212503

**Reply by VlKlNGS** [reply]
But I already have the -platform xcb argument added to the desktop file and this is how it behaves... :(
Is this not correct, or do I have to add something extra?

![keepassxcDesktopFile](https://github.com/user-attachments/assets/7a78dbd1-d8c2-4e3b-8401-00b94c0978b3)

**Reply by droidmonkey** [reply, marked as answer]
The dialog you show is not from KeePassXC, it is from KDE, you'll need to consult KDE's  documentation or forums

**Reply by VlKlNGS** [reply]
I see, ok I will try that too. Thank you for your help.
